supply-chain/config.toml

```diff
@@ -255,14 +271,18 @@ criteria = "safe-to-deploy"
 version = "0.8.20"
 criteria = "safe-to-deploy"
 
-[[exemptions.crunchy]]
-version = "0.2.3"
-criteria = "safe-to-deploy"
-
 [[exemptions.crypto-common]]
 version = "0.2.0-rc.1"
 criteria = "safe-to-deploy"
 
+[[exemptions.curve25519-dalek]]
+version = "4.1.0"
+criteria = "safe-to-deploy"
```

Comment:
There are a lot of exemptions for cryptographic dependencies being added here. Should we try to review some of them?

Reply:
Maybe at some point, but currently we don't have time. My general approach for these is to exempt the first time a dependency lands, and then attempt to review diffs as they occur if they are small enough.